### PR TITLE
 Ports "Mime PDAs only accept emojis for messages, mime cartridge comes with an emoji guidebook"

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Tgstation PRs #47177, #47420 and #47542.
Scrapping my older PDA emojis feature in favor of turning the allow_emojis var on by default in the current one.

## Why It's Good For The Game
Making the mime PDAs compliant with the vow of silence.

## Changelog
:cl: tralezab, bandit, Skoglol
add: The mime's PDA messages are silent now!
add: 30 new emoji have been added. Mime buff, mime now OP.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Finally, some good fuckin' code.

**Review by silicons — APPROVED** [review]

**Comment by ForrestWick** [comment]
The mime being able to use their PDA is for emergencies. This literally just removes a vital form of communication from them in the event they need the AI, a head/sec, or wanna talk to friends casually from a distance. There is no reason for this PR to exist and equally so nothing it contributes to make the game more enjoyable

**Comment by Ghommie** [comment]
You can always bug the HoP, a cyborg or anyone with the required access to the PDA vendor to buy you another PDA as soon as the round start.
I could add a station red alert level check to skip the restriction later though.

**Comment by Linzolle** [comment]
honestly the best part of this PR is that it replaces "flushed" with the actual emoji instead of the fuckin toilet

**Comment by ghost** [comment]
@kevinz000 merge u wont
:nya: :peel: :silentman: :joy:

File: code/game/machinery/telecomms/machines/message_server.dm [hunk 1]
```diff
@@ -107,10 +107,9 @@
 	return data["targets"][1]
 
 /datum/signal/subspace/pda/proc/format_message(emojify = FALSE)
```
Comment (Linzolle) [inline code/game/machinery/telecomms/machines/message_server.dm]: unused variable emojify

**Comment by Ghommie** [comment]
> The usual "space_cola" issue.

 :unamused:

**Review by Linzolle — APPROVED** [review]

**Comment by YakumoChen** [comment]
>complaining Mime is harder because it cant use PDA to not be silent

**Pull request merged**